src/transactionNotifier/TransactionNotifierView.tsx

```diff
@@ -82,14 +86,14 @@ export function describeTxStatus(tx: TxState) {
     case TxStatus.Propagating:
       return (
         <>
-          Propagating <Timer start={tx.broadcastedAt} />
+          Unconfirmed <Timer start={tx.broadcastedAt} />
         </>
       );
     case TxStatus.WaitingForConfirmation:
       return (
-        <>
+        <span className={styles.hilight}>
           Unconfirmed <Timer start={tx.broadcastedAt} />
```

Comment:
Shouldn't link to etherscan be provided here? That was @geronimo's idea afaik.

Reply:
That was not requested in E2D-396

Reply:
Now it is. ;-)